core/dispatcher/src/main/scala/whisk/core/dispatcher/ActivationFeed.scala

```diff
@@ -83,7 +84,12 @@ protected class ActivationFeed(
                             handler(topic, bytes)
                     }
                 }
-                consumer.commit()
+                Try {
+                    consumer.commit()
+                } recover {
+                    case e: CommitFailedException => logging.warn(this, s"Error on committing to Kafka ${e.getMessage}")
+                    case e: Throwable             => logging.error(this, s"Unexpected error on committing to Kafka: ${e.getMessage}")
+                }
                 fill()
```

Comment:
we need to test this - in particular, the test should kick the consumer out of the group and make sure the kicked out consumer can re-establish connection for a new poll.